doc/01_Pattern interazione/07_accesso-crud-a-risorse.rst

```diff
@@ -3,16 +3,16 @@ Accesso CRUD a risorse
 
 In un’architettura orientata alle risorse le API vengono utilizzate, più
 che per eseguire compiti complessi, per eseguire operazioni di tipo CRUD
-- Create, Read, Update, Delete su risorse del dominio di interesse. Ad
+- Create, Read, Update, Delete - su risorse del dominio di interesse. Ad
 esempio, una prenotazione è una risorsa che può essere creata (quando
 viene fissata), letta, modificata ed eliminata.
 
-In questo scenario si assume che le API sono utilizzare per la gestione
-da parte del fruitore delle risorse messe a disposizione dell’erogatore.
-L’insieme di operazione CRUD garantire dall’erogatore dipende dalla
-natura della risorse e dalla relazione costruita con i fruitori, sono
-possibili relazione in cui l’erogatore rende disponibile ai fruitori la
-solo operazione di lettura (Read).
+In questo scenario si assume che le API siano utilizzare per la gestione
```

Comment:
sostituirei "utilizzare" in "utilizzate", era un problema già presente

Reply:
Sì sono d'accordo

Reply:
```suggestion
In questo scenario si assume che le API siano utilizzate per la gestione
```